modules/mplayer.yaml

```diff
@@ -0,0 +1,16 @@
+name: mplayer
```

Comment:
this is not auto generated and should be part of the main manifest

Reply:
I see that this module mplayer also contains submodule yasm.
If I create a module entry for each of them - in a simillar fashoin yasm is described in mplayer manifest but in the main manifest for the app will that be acceptable?

Reply:
It's generally better to include everything into a single manifest.